starter-code/styles/modules.css

```diff
@@ -55,3 +57,11 @@
          - Main should be all on one row, with padding spaced better for a larger screen.
          - Just, ya know, try to make it look good! */
 /* @media only screen... */
+@media only screen and (min-width: 640px) {
+  .main-nav ul, li {
```

Comment:
Good job making the contents of the nav appear! Be careful changing all the `li` at the same time. This will change all the `li` on the page to `display: block`, even the ones outside the nav.